src/compiler/python_generator.cc

```diff
@@ -101,6 +101,11 @@ PrivateGenerator::PrivateGenerator(const GeneratorConfiguration& config,
 void PrivateGenerator::PrintAllComments(StringVector comments,
                                         grpc_generator::Printer* out) {
   if (comments.empty()) {
+    // It's important we print something here and not omit this line,
```

Comment:
Probably only if both are missing. I only tested with the empty no-comment service, but I presume it'd print a comment if one existed and makes python happy. In any case, reworded such that such confusion would not arise.